### DISCUSSION TITLE
asdf plugin for V

### DISCUSSION THREAD

**Discussion by x80486** [vlang/v#19151] [Questions and Answers]
Would it be possible for you to help on maintaining the `asdf` [plugin](https://github.com/jthegedus/asdf-v) for `V`? I think it should be straightforward, but it's been broken for a while.

There are pretty good reasons to use `asdf` to install, specifically, programming languages, run-times, and software development kits, because it makes it straightforward to maintain several versions.

**Comment by jlmess77** [comment]
I would start with this: https://github.com/jthegedus/asdf-v/blob/main/CONTRIBUTING.md When you made some changes, you can send some pull requests with your patches.
